src/doc/rustc/src/platform-support.md

```diff
@@ -266,6 +266,7 @@ target | std | host | notes
 `msp430-none-elf` | * |  | 16-bit MSP430 microcontrollers
 `powerpc-unknown-linux-gnuspe` | ✓ |  | PowerPC SPE Linux
 `powerpc-unknown-linux-musl` | ? |  |
+`powerpc-unknown-linux-muslspe` | ? |  | PowerPC SPE Linux
```

Comment:
Can you make this a link to the new file?

```suggestion
[`powerpc-unknown-linux-muslspe`](platform-support/powerpc-unknown-linux-muslspe.md) | ? |  | PowerPC SPE Linux
```

Reply:
Oh, right. Done. Thanks.